.github/workflows/cd.yml

```diff
@@ -0,0 +1,47 @@
+name: CD
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build-and-deploy:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+        fetch-depth: 0
+
+# Compile and run tests
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+      working-directory: source
+    - run: npm run build
+      working-directory: source
+    - run: npm test
+      working-directory: source
+
+# Push dist/ to dist
+    - run: git checkout --orphan dist
```

Comment:
Why not `gh-pages`? I think this is the default, and as long as we don't have any other purposes for this branch, `gh-pages` would be the most descriptive.